components/core/JsInterop/ObservableApi/resizeObserver.ts

```diff
@@ -1,8 +1,10 @@
 ﻿import { getDom } from '../interop';
 
+// @ts-ignore
```

Comment:
May I ask what errors were you getting?

Reply:
> Error TS2304: Cannot find name 'ResizeObserver'. 

Maybe because it's experimental? (In fact, all modern browsers support it except IE)

Reply:
I don't think it is experimental any longer. I cannot find any reference to it being experimental... We do have a detection in place if it does not exist and a fallback to window.resize. On the other hand, I think IE will [not be supported by AntBlazor anylonger.](https://github.com/ant-design-blazor/ant-design-blazor/pull/1392#issuecomment-825404976)

To be honest I never got that error. But I guess there is no harm to have it here though 👍🏻 .

Reply:
Yes, the word "experimental" does not exist in the English document of MDN. Maybe the Chinese document of MDN has not been updated.